### DISCUSSION TITLE
30 B

### DISCUSSION THREAD

**Discussion by GitHub1712** [oobabooga/text-generation-webui#1319] [Q&A]
Someone running a 30B model? Which is cool?

I am very impressed about the eachadea/vicuna-13b-1.1 model. You can simple install:
    Start download-model.bat, tyle L for custom model, insert eachadea/vicuna-13b-1.1
    After download you should be able to select the model from the web ui.
    Alternative you could download using the web ui from the model tab.

**Comment by OWKenobi** [comment]
I used to run llama-30b quantized, it was kinda cool. At the moment I lack good benchmarks, sometimes, I think I like a model, but later find out it is bad in another task or so. Or a model I don't like turns out doing great with a right prompt. So just try it yourself and tell me what you think!
